xml/System.Security.Cryptography.X509Certificates/X509CertificateCollection.xml

```diff
@@ -1042,8 +1042,8 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets a value indicating whether the collection has a fixed size.</summary>
+        <value><see langword="true"/> if the collection has a fixed size; otherwise, <see langword="false"/>. The default is <see langword="false"/>.</value>
         <remarks>To be added.</remarks>
```

Comment:
```suggestion
         <remarks>
           <format type="text/markdown"><![CDATA[

## Remarks

This member is an explicit interface member implementation. It can be used only when the <xref:System.Security.Cryptography.X509Certificates.X509CertificateCollection> instance is cast to an <xref:System.Collections.IList> interface.  

           ]]></format>
         </remarks>
```